ensemble/ctf/tests/test_editor.py

```diff
@@ -33,7 +35,8 @@ def setUp(self):
     EnableCanvas:
         component << editor
 """
-
+        if six.PY2:
+            enaml_source = enaml_source.encode('utf-8')
```

Comment:
Does enaml's parser really expect input in the whatever `str` maps to? Or can you pass utf-8 bytes on both platforms?

(we could also think about fixing this in `traits_enaml` to make life easier)

Reply:
I can't pass utf-8 bytes to enaml's parser on Python 3. For more info, see https://github.com/enthought/ensemble/pull/66#issuecomment-493005807

Reply:
That's unfortunate... OK.